files/usr/share/jeos-firstboot/modules/ssh_enroll

```diff
@@ -0,0 +1,131 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: Copyright 2024 SUSE LLC
+# shellcheck shell=bash
+
+ssh_enroll_handle_pubkeys()
+{
+	filename="$1"
+	if ! [ -s "$filename" ]; then
+		d --msgbox $"No public keys received!" 0 0
+		return 0
+	fi
+
+	items=()
+	lineno=1
+	state=on
+	while read -r type key comment; do
+		items+=("$lineno" "${type} ${key:0:8}...${key: -20:20}" "$state")
+		lineno=$((lineno+1))
+		state=off # Only import the first key by default
```

Comment:
Why import the first key by default? Is there some kind of priority ordering in SSH clients that makes the first key more fitting than the later ones?

I would not set any default here. Credentials are too sensitive to make any conclusions. The user should explicitly select the ones he wants.

Reply:
> Why import the first key by default? Is there some kind of priority ordering in SSH clients that makes the first key more fitting than the later ones?

Yes, three reasons:

* It's the order defined in the ssh config file, i.e. by the user
* The first key is tried first, i.e. it's slightly quicker to log in
* If an SSH agent is used, the agent provided keys are first, i.e. the key the user most likely wants to use

Reply:
Ok, first key is no longer selected by default